assembl/static2/css/views/synthesis.scss

```diff
@@ -1,3 +1,18 @@
+.syntheses-page {
+  .syntheses-no-synthesis {
+    font-size: 20px;
```

Comment:
try to avoid hard coded font size and color since there are variables for that. please check variables.scss > $font-size-***

Reply:
k to be replaced with $font-size-m